main/inc/lib/userportal.lib.php

```diff
@@ -1920,6 +1920,18 @@ public function returnCoursesAndSessions(
 
                                             $course_session['extrafields'] = CourseManager::getExtraFieldsToBePresented($course['real_id']);
 
+                                            if (api_get_configuration_value('enable_unsubscribe_button_on_my_course_page')
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line exceeds maximum limit of 120 characters; contains 122 characters](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5478348)